From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Mon, 1 Apr 2024 21:15:03 -0600
Subject: [PATCH] kernelbase: HACK: Force in-process-gpu for Summer Islands.

CW-Bug-Id: #23635
Valve-commit: 44fd0130497cc3b774901d858f138478cf0ed541
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index 2715f1b..ff09d51 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -511,6 +511,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
     }
     options[] =
     {
+        {L"SummerIslands.exe", L" --in-process-gpu"},
         {L"Paradox Launcher.exe", L" --use-angle=gl"},
         {L"Montaro\\nw.exe", L" --use-gl=swiftshader"},
         {L"Aisling and the Tavern of Elves\\nw.exe", L" --use-gl=swiftshader"},
